### PR TITLE
Add support for Visual Studio 2019

### LINKED ISSUE #9: Release extension to VS2019 Enterprise Preview
This extension is sooo useful 😄!
I stopped using VS2019 Enterprise Preview because some extensions were not released to it.

### PR DESCRIPTION
**Pull request by mniak**
Closes #9 

This PR covers the following changes

- Widen range of `InstallationTarget`
- Widen range of `Prerequisite`
- Bump version one minor unit

### CODE REVIEW

**Comment by pauer24** [comment]
Dear mniak, thanks a lot for your contribution!!!

Unfortunately, I cannot approve it, since it does allow the installation on VS2019 RC. The current version I'm using of the RC, is 16.0.0 RC.2 .

Did you tried it and worked? Which version were you using?

Again,thanks a lot.

P.

**Comment by pauer24** [comment]
BTW,  after updating the source.extension.vsixmanifest, it still does not work...
image

**Comment by spottedmahn** [comment]
Reading [this article](https://devblogs.microsoft.com/visualstudio/how-to-upgrade-extensions-to-support-visual-studio-2019/), maybe the `<Prerequisite>` has to be open ended?  I woudln't think so but maybe?

image

**Comment by pauer24** [comment]
Dear contributors, I already achieved that the installer allowed me to install the extension on VS2019. Unfortunately, after installing it, it was not available anywhere in the VS... not even in the extensions section, though it didn't allow me to reinstall it again... So extrange!!
Any of you have any clue on what can be happening?

File: NamespaceFixer/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,20 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="pch9d11e-9ed1-48f6-875b-4d144a5d790c" Version="1.5" Language="en-US" Publisher="pcerv" />
+        <Identity Id="pch9d11e-9ed1-48f6-875b-4d144a5d790c" Version="1.6" Language="en-US" Publisher="pcerv" />
         <DisplayName>NamespaceFixer</DisplayName>
         <Description xml:space="preserve">VS Extension to fix namespaces following C# convention</Description>
         <Icon>Package.ico</Icon>
     </Metadata>
     <Installation>
```
Comment (spottedmahn) [inline NamespaceFixer/source.extension.vsixmanifest]: There's two `InstallationTarget`'s.  Maybe that's the problem?

I removed the first one and it's working via the debugger in VS 2019

image


image

Comment (spottedmahn) [reply NamespaceFixer/source.extension.vsixmanifest]: Seems to work 😊

image

[I finally figured out how to create the installer 😊](https://docs.microsoft.com/en-us/visualstudio/extensibility/walkthrough-publishing-a-visual-studio-extension?view=vs-2017#package-your-extension)

Comment (spottedmahn) [reply NamespaceFixer/source.extension.vsixmanifest]: @pauer24 (not sure if you're notified automatically or not?)

**Comment by pauer24** [comment]
Hello mniak, I've already approved a PR that I could test successfuly.
I'm so sorry to approve the other one before yours. I just realized (after approving), that the checks I made with your PR were completly wrong, and that's the main reason it didn't work for me...
Anyway, I want to specially thank you for your effort, and I hope you will do tons of more PR.
Regards,

P.

**Comment by mniak** [comment]
No problem, @pauer24.
The important thing is that the fix/update was made.
Thanks for the nice project. ✋

**Pull request closed without merging**